Emby.Server.Implementations/LiveTv/EmbyTV/RecordingHelper.cs

```diff
@@ -48,12 +49,19 @@ public static string GetRecordingName(TimerInfo info)
 
                 if (!string.IsNullOrWhiteSpace(info.EpisodeTitle))
                 {
+                    var tmpName = name;
                     if (addHyphen)
                     {
-                        name += " -";
+                        tmpName += " -";
+                    }
+                    // Calculate the length of the resulting filename
+                    var recordingNameLength = Encoding.UTF8.GetByteCount(tmpName) + Encoding.UTF8.GetByteCount(info.EpisodeTitle);
+                    //  Since the filename will be used with file ext. (.mp4, .ts, etc)
+                    if (recordingNameLength < 250)
+                    {
+                        tmpName += " " + info.EpisodeTitle;
```

Comment:
The string you're calculating the size of doesn't include the space before the title. That might have been intended by the 250 max size selection but that's not obvious. Why not simplify it a bit:
```suggestion

                    tmpName += " " + info.EpisodeTitle;
                    //  Since the filename will be used with file ext. (.mp4, .ts, etc)
                    if (Encoding.UTF8.GetByteCount(tmpName) < 250)
                    {
```